audio_common_msgs/AudioDataStamped audio
---
infrastructure/EmoProbArr emotion
